compiler/src/dotty/tools/dotc/transform/Bridges.scala

```diff
@@ -16,7 +16,10 @@ class Bridges(root: ClassSymbol, thisPhase: DenotTransformer)(implicit ctx: Cont
 
   assert(ctx.phase == ctx.erasurePhase.next)
   private val preErasureCtx = ctx.withPhase(ctx.erasurePhase)
-  private val elimErasedCtx = ctx.withPhase(ctx.elimErasedValueTypePhase.next)
+  private def elimErasedCtx = ctx.withPhase(
```

Comment:
I think I would just make it a lazy val and let it fail if Ystop-after:erasure is true, this is less bad than doing something slightly different which might be confusing when debugging an issue. (even better might be to tweak Ystop-after so that the phase still exists, even if we stop before it).

Reply:
Done